Documentation/Usage.md

```diff
@@ -279,31 +277,38 @@ This is because `jpeglib.h` is not a correct module, that is, it does not contai
 the required line `#include <stdio.h>`. Alternatively, you can add `#include <stdio.h>`
 to the top of jpeglib.h to avoid creating the `shim.h` file.
 
-Create a Git repository and tag it:
-
-    CJPEG$ git init
-    CJPEG$ git add .
-    CJPEG$ git commit -m "Initial Commit"
-    CJPEG$ git tag 1.0.0
-
 Now to use the CJPEG package we must declare our dependency in our example
 app’s `Package.swift`:
 
 ```swift
+// swift-tools-version: 5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
 import PackageDescription
 
 let package = Package(
     name: "example",
-    dependencies: [
-        .package(url: "../CJPEG", from: "1.0.0")
+    targets: [
+        // Targets are the basic building blocks of a package, defining a module or a test suite.
+        // Targets can depend on other targets in this package and products from dependencies.
+        .executableTarget(
+            name: "example",
+            dependencies: ["CJPEG"],
+            path: "Sources"
+            ),
+        .systemLibrary(
```

Comment:
ditto re. splitting the example into two steps

Reply:
After splitting the first one, I'm not so sure if we should split this one as well. It would add another blob of code, but since the second example builds up on the first one, I think the concept should be clear even without splitting. 

But, happy to split if you'd prefer it in two steps here as well!